testsuite/performance/tests/pom.xml

```diff
@@ -29,44 +29,56 @@
     <artifactId>performance-tests</artifactId>
     <name>Keycloak Performance TestSuite - Tests</name>
 
-    <description>
-    </description>
-
     <properties>
-        <compose.file>docker-compose.yml</compose.file>
-        <compose.up.params/>
-        <compose.restart.params>keycloak</compose.restart.params>
-        <keycloak.server.uris>http://localhost:8080/auth</keycloak.server.uris>
-        <db.url>jdbc:mariadb://keycloak:keycloak@localhost:3306/keycloak</db.url>
+        <provisioner>docker-compose</provisioner>
+        <deployment>singlenode</deployment>
 
+        <!-- Keycloak Server Settings -->
+        <keycloak.scale>2</keycloak.scale>
+        <keycloak.dc1.scale>1</keycloak.dc1.scale>
+        <keycloak.dc2.scale>1</keycloak.dc2.scale>
+        <keycloak.docker.cpuset>2-3</keycloak.docker.cpuset>
```

Comment:
Having two properties of almost the same name is too error-prone: `keycloak.docker.cpuset` vs `keycloak.docker.cpusets`. Can't we use `keycloak.docker.cpusets` only?

Reply:
Yes, this can be simplified.